api/envoy/api/v2/core/grpc_service.proto

```diff
@@ -78,28 +152,8 @@ message GrpcService {
   // request.
   google.protobuf.Duration timeout = 3;
 
-  // gRPC credentials as described at
-  // https://grpc.io/docs/guides/auth.html#credential-types.
-  //
-  // .. note::
-  //
-  //   Credentials are only currently implemented for the Google gRPC client.
-  message Credentials {
-    oneof credential_specifier {
-      option (validate.required) = true;
-
-      // OAuth2 access token, see
-      // https://grpc.io/grpc/cpp/namespacegrpc.html#ad3a80da696ffdaea943f0f858d7a360d.
-      string access_token = 1;
-      // [#comment: TODO(htuch): other gRPC auth types, e.g. IAM credentials, JWT, etc.]
-    }
-  }
-  // A set of credentials that will be composed to form the `channel credentials
-  // <https://grpc.io/docs/guides/auth.html#credential-types>`_.
-  repeated Credentials credentials = 4;
-
   // Additional metadata to include in streams initiated to the GrpcService.
   // This can be used for scenarios in which additional ad hoc authorization
   // headers (e.g. `x-foo-bar: baz-key`) are to be injected.
-  repeated HeaderValue initial_metadata = 5;
+  repeated HeaderValue initial_metadata = 4;
```

Comment:
These are breaking config changes. I don't think this is OK? @htuch?

Reply:
IMO it's not OK at this point to change the outer GrpcService message. I think changing the inner Google config is probably OK.

Reply:
Ok, that makes sense.  However, for the credentials field that was previously number 4, I assume I can remove that and mark field 4 as reserved?

Reply:
Yup that's fine.

Reply:
Yeah, I'd try and minimize the breakage and just strip the credentials, which nobody is using, without renumbering.

Reply:
By nobody I mean nobody other than Google and a handful of other folks who understand that this is evolving :) We can deal with the breakage here right now.